pkg/controller/datavolume-controller.go

```
@@ -278,6 +278,26 @@ func (r *DatavolumeReconciler) Reconcile(req reconcile.Request) (reconcile.Resul
 	return r.reconcileDataVolumeStatus(datavolume, pvc)
 }
 
+func (r *DatavolumeReconciler) findPVCForDV(dataVolumeNamespace string, dataVolumeName string) (*corev1.PersistentVolumeClaim, error) {
+	labelSelector, err := labels.Parse(fmt.Sprintf("cdi-controller=%v", dataVolumeName))
+	if err != nil {
+		return nil, err
+	}
+
+	pvcList := &corev1.PersistentVolumeClaimList{}
+	err = r.client.List(context.TODO(), pvcList, &client.ListOptions{Namespace: dataVolumeNamespace, LabelSelector: labelSelector})
+	if err != nil {
+		return nil, err
+	}
+
+	if len(pvcList.Items) > 0 {
+		return &pvcList.Items[0], nil
+	}
+	// What if there's more than one? This should not be possible, but stuff happens?
```

If we find more than one, that is a problem, and we should return an error.